### PR TITLE
feat: likes page

### PR DESCRIPTION
**Pull request by tusharkandpal**
To Check:

- `likeSlice.js` for liked videos
- added `Likes.jsx` page
- added `getLikedVideos.js`, `addToLikedVideos.js` & `removeFromLikedVideos.js` services
- implemented like/dislike functionality in `VideoCard.jsx` & `Video.jsx`

### CODE REVIEW

File: src/features/authSlice.js [hunk 1]
```diff
@@ -3,22 +3,22 @@ import { logInService, signUpService } from "../services/services";
...
+  async (credentials, { rejectWithValue, dispatch }) => {
+    return await logInService(credentials, rejectWithValue, dispatch);
   }
 );
 
 export const signUp = createAsyncThunk(
   "auth/signUp",
-  async (credentials, { rejectWithValue }) => {
-    return await signUpService(credentials, rejectWithValue);
+  async (credentials, { rejectWithValue, dispatch }) => {
```
Comment (Kothalkarpriya) [inline src/features/authSlice.js:20]: GOod you are using async and await

**Review by Kothalkarpriya — APPROVED** [review]
The code is clear and understandable. Good Work

File: src/services/auth/logInService.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+      addToast({
+        type: "SUCCESS",
+        desc: `Successfully Logged-In. Welcome <strong>${data.foundUser.firstName}</strong> 👋 !`,
+      })
+    );
     return data;
   } catch (error) {
-    return rejectWithValue(`Error Logging In. Please try again !`);
+    dispatch(
+      addToast({
```
Comment (Megha-Pathak) [inline src/services/auth/logInService.js:23]: Good thinking for adding toast here.

**Review by Megha-Pathak — APPROVED** [review]

**Review by Megha-Pathak — APPROVED** [review]
Rest of the code looks moduler and well structured. The video card and like are working as intended.

**Pull request merged**